third_party/xla/xla/service/gpu/llvm_gpu_backend/gpu_backend_lib.cc

```diff
@@ -854,7 +854,10 @@ absl::StatusOr<std::vector<uint8_t>> EmitModuleToHsaco(
   }
   // Locate lld.
   std::string lld_path;
-  if (std::getenv("ROCM_PATH")) {
+  if (std::getenv("LLVM_PATH")) {
+       lld_path = tsl::io::JoinPath(std::getenv("LLVM_PATH"), "bin");
+  }
+  else if (std::getenv("ROCM_PATH")) {
        lld_path = tsl::io::JoinPath(std::getenv("ROCM_PATH"), "llvm/bin");
```

Comment:
I think we need to upstream these XLA change as well.

Reply:
Sounds good, I created a pr on xla as well: https://github.com/ROCm/xla/pull/28

Reply:
:) thanks! in XLA we're fully working on upstream, so please send PR to https://github.com/openxla/xla but I think it will ask you to verify some auth stuff, let me cherry-pick your change to do it.

Reply:
@i-chaochen  not sure cherry-picking would also work until @afzpatel has signed CLA. We may have to submit these changes as ourselves and credit @afzpatel  in the commit message

Reply:
I just realise it has some tsl changes that cannot send PR to upstream...e.g., this one https://github.com/openxla/xla/pull/13879 we need to ask upstream to manually add it...but let's have this PR against upstream first.

Reply:
https://github.com/openxla/xla/pull/14854 done @afzpatel